styles.css

```diff
@@ -0,0 +1,155 @@
+* {
+  margin: 0;
+  padding: 0;
+  font-family: "Poppins", sans-serif;
+  box-sizing: border-box;
+  text-decoration: none;
+}
+
+html {
+  scroll-behavior: smooth;
+}
+
+.header {
+  top: 0;
+  left: 0;
+  width: 100%;
+  padding: 20px 10%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  flex-wrap: wrap;
+  background-color: #000;
+  opacity: 0.8;
+}
+
+ul {
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: row;
+}
+
+body {
+  background-image: url(Lantern.JPG);
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  height: 100vh;
+  background-color: rgb(178, 177, 177);
+}
+
+.logo {
+  font-size: 35px;
+  color: #fff;
+  font-weight: 500;
+}
+
+.header nav a {
+  font-size: 20px;
+  color: #fff;
+  font-weight: 350;
+  margin-left: 35px;
+  transition: 0.5s;
+}
+
+.header nav a:hover {
+  background-color: orange;
+  padding: 0.8rem 0.8rem;
+  border-radius: 5px;
+}
+
+.main {
+  background-color: #fff;
+  opacity: 0.9;
+  margin-top: 25rem;
+  margin-bottom: 25rem;
+  padding-left: 10rem;
+}
+
+span {
+  color: rgb(112, 26, 26);
+}
+
+.developer {
+  font-size: 25px;
+  color: rgb(87, 47, 47);
+  font-weight: bold;
+}
+
+.description1 {
```

Comment:
the naming of classes can be something different than description1, description2. F.x. when you will be working in a team, it can be confusing..